### DISCUSSION TITLE
Uncaught Error: Cannot find module 'components/ui/Switch'

### DISCUSSION THREAD

**Discussion by SmartManoj** [chatwoot/chatwoot#1527] [Q&A]
![image](https://user-images.githubusercontent.com/7231077/102592173-1bc88700-4139-11eb-9982-014a3327e959.png)

**Comment by pranavrajs** [comment]
@SmartManoj Did you change any code? I couldn't see this error in our apps.

**Reply by SmartManoj** [reply]
changed "@rails/webpacker": "rails/webpacker#b6c21802",

**Reply by SmartManoj** [reply]
In Windows 10 without WSL

**Reply by pranavrajs** [reply]
Why is this changed?

**Reply by SmartManoj** [reply]
On fresh installation got, https://del.dog/bug_in_node-gyp.txt

**Comment by SmartManoj** [comment]
Should we use issues or discussions for this Q/A?

**Reply by pranavrajs** [reply]
If there is a bug in the exisitng code without changes, you can create issues. Otherwise, a discussion is a better fit.

**Comment by SmartManoj** [comment]
[webpackMissingModule: Cannot find module 'components/ui/Switch'](https://stackoverflow.com/q/65356909/5033247)

**Comment by SmartManoj** [comment]
NVM, run in test env; How should I close this?
